src/pieces/utils.py

```diff
@@ -35,7 +27,7 @@ def extract_code_from_markdown(markdown, name, language):
 
     # Writing the extracted code to a new file
     with open(file_path, 'w') as file:
-        file.write(extracted_code)
+        file.write(code)
 
     return file_path
 
```

Comment:
Ok, so are you using an endpoint in the SDK to extract the code from markdown instead of BS now?

Reply:
Yupp the returned text is not md it is raw string I am sure ik there is a markdown but no need to extract it!